spec/support/shared_examples.rb

```diff
@@ -1,6 +1,6 @@
 require 'pathname'
 
-shared_examples_for "an rspec-rails example group mixin" do |type, *paths|
+RSpec.shared_examples_for "an rspec-rails example group mixin" do |type, *paths|
```

Comment:
It seems that we require spec support files before `RSpec.configure`, and `Core::DSL.expose_globally!` was taking precedence. Interesting.

Reply:
Sounds about right